bolt_unix.go

```diff
@@ -58,7 +58,12 @@ func mmap(db *DB, sz int) error {
 	}
 
 	// Map the data file to memory.
-	b, err := syscall.Mmap(int(db.file.Fd()), 0, sz, syscall.PROT_READ, syscall.MAP_SHARED)
+	fv := syscall.MAP_SHARED
+	if db.MmapFlags != 0 {
+		// OR on each mmap() call
+		fv = syscall.MAP_SHARED | db.MmapFlags
+	}
```

Comment:
You don't need to check if `MmapFlags` is `0` since OR'ing a zero value won't do anything. You can just do:

```
b, err := syscall.Mmap(int(db.file.Fd()), 0, sz, syscall.PROT_READ, syscall.MAP_SHARED | db.MmapFlags)
```